src/views/AddItem.jsx

```diff
@@ -1,4 +1,5 @@
 import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
 import { addItem } from '../api/firebase';
 
 // sets number of days for each future purchase estimate variable
```

Comment:
@tmflower This is part of this PR but remove the magic number on line 13. Let declare as
 ```javascript
const initialState = { itemName: '', estimate: numDaysInEstimate.soon};
```